recipe/meta.yaml

```diff
@@ -1,31 +1,22 @@
 # Some useful notes can be found at https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT
 
-{% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
-
-# The version should be in the form X.Y, where X denotes major infrastructural change to how blas is managed,
-# and Y denoting blas priority. Currently Y=0 not default, Y=1 default blas.
-{% set Y = 1 if variant == 'openblas' else 0 %}
 package:
   name: blas
-  version: 1.{{ Y }}
+  version: 2.0
 
 build:
   # We mustn't use the build number in the blas package - because we are defining the string,
   # manually we would not be producing unique filenames.
   number: 0
-  string: {{ variant }}
-  track_features:
-    # The "blas" package must always track a feature, else it will be
-    # prioritised over all others.
-    - blas_{{ variant }}
+  string: {{ blas }}
+  run_exports:
+    - {{ pin_subpackage("blas", exact=True) }}
```

Comment:
Can we do something like `blas=2.*=openblas`? Otherwise we'll be pinning to an exact package and we can't make any change. (For eg: add a license_file entry)

Reply:
:+1: done

Reply:
Seems like the current way to actually get these in is

```
run_exports
    - blas >=2.0 {{ blas }},<3.0a {{ blas }} 
```

Reply:
updated